src/Entity/BaseGallery.php

```diff
@@ -27,6 +28,28 @@ public function __construct()
         $this->galleryHasMedias = new ArrayCollection();
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function setGalleryHasMedias($galleryHasMedias)
+    {
+        $this->galleryHasMedias = $galleryHasMedias;
+        foreach ($this->galleryHasMedias as $galleryHasMedia) {
+            $this->addGalleryHasMedias($galleryHasMedia);
```

Comment:
Why do you need this? Shouldn't be replaced by simply: `$galleryHasMedia->setGallery($this);` ?

Reply:
Do you want to remove addGalleryHasMedias method?

Reply:
If we don't need it yes